builds/serverless-config-pack/serverless-go.yml

```diff
@@ -0,0 +1,111 @@
+service: ${file(deployment-env.yml):service}
```

Comment:
this will have to be updated as per what we have in events branch. there have been some changes to other serverless files